README.md

```diff
@@ -1,102 +1,105 @@
 
-⚠ This plugin is under development, API and CLI might change! ⚠
+Certbot plugin to obtain TLS certificates from Let's Encrypt for domains hosted with deSEC.io, using the DNS challenge
+challenge mechanism.
 
-[deSEC](https://desec.io/) DNS Authenticator plugin for [Certbot](https://certbot.eff.org/)
 
-This plugin automates the process of completing a ``dns-01`` challenge by
-creating, and subsequently removing, TXT records using the deSEC DNS API.
+## Installation
 
-## Configuration of deSEC
+To get certificates from Let's Encrypt, install certbot and this plugin.
+There are many ways to install certbot, this guide uses Python's `pip`:
 
-Log in at [deSEC](https://desec.io/) and create the domain you would like to use and a token for DNS management.
+```
+python3 -m pip install certbot certbot-dns-desec
+```
 
-## Installation
+## Prerequisites
 
-This certbot plugin can be installed using `pip`:
+To get a Let's Encrypt certificate for your domain,
+you need an access token for the deSEC.io account that holds the domain you need the certificate for.
+Also make sure that your domain name has been delegated to deSEC according to deSEC's instructions.
 
-    python3 -m pip install certbot-dns-desec
 
+## Issue Certificate
 
-## Command Line Interface
+To issue and renew certificates using `certbot-dns-desec`, an access token to your deSEC account is required.
+To store such a token in a secure location, use, e.g.:
 
-To start using DNS authentication with deSEC, pass the following arguments on
-certbot's command line:
+```
+DOMAIN=example.com
+TOKEN=your-desec-access-token
+sudo mkdir /etc/letsencrypt/secrets/
+sudo chmod 700 /etc/letsencrypt/secrets/
+echo "dns_desec_token = $TOKEN" | sudo tee /etc/letsencrypt/secrets/$DOMAIN.ini
+sudo chmod 600 /etc/letsencrypt/.secrets/$DOMAIN.ini
+```
 
-1. ``--authenticator dns-desec`` Selects the authenticator plugin.
-1. ``--dns-desec-credentials <file>`` Specifies the file holding the deSEC API credentials (required, see below).
-1. ``--dns-desec-propagation-seconds`` Waiting time for DNS to propagate before asking the ACME server to verify the
-    DNS record (default 5).
+Adjust `$DOMAIN` and `$TOKEN` according to your domain and deSEC access token, respectively.
+The file location is just a suggestion and can be changed.
 
+With the credentials stored, you can apply for a wildcard certificate for your domain by using, e.g.,
 
-## deSEC API Credentials
+```
+certbot certonly \
+     --authenticator dns-desec \
+     --dns-desec-credentials /etc/letsencrypt/secrets/$DOMAIN.ini \
+     -d "$DOMAIN" \
+     -d "*.$DOMAIN"
+```
 
-An example ``credentials.ini`` file:
+In this command, `--authenticator dns-desec` activates the `certbot-dns-desec` plugin;
+the `--dns-desec-credentials` argument provides the deSEC access token location to the plugin.
+These flags can be combined with more sophisticated usages of certbot, e.g. to automatically reload servers after the
+renewal process.
```

Comment:
... `Such functionality is independent of this plugin; for details, see the certbot documentation.`